specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2017-12-01-preview/postgresql.json

```diff
@@ -1579,13 +1688,35 @@
           "$ref": "#/definitions/SslEnforcement",
           "description": "Enable ssl enforcement or not when connect to server."
         },
+        "minimalTlsVersion": {
+          "$ref": "#/definitions/MinimalTlsVersion",
+          "description": "Enforce a minimal Tls version for the server."
+        },
+        "byokEnforcement": {
+          "type": "string",
+          "description": "Status showing whether the server enabled for customer managed encryption key.",
+          "readOnly": true
+        },
+        "infrastructureEncryption": {
+          "type": "string",
+          "description": "Enable Storage Auto Grow.",
```

Comment:
same comment as above - ok if this is an official marketing name for this featrue, other wise should explain 'auto grow'

Reply:
yes this is indeed the official marketing name : https://docs.microsoft.com/en-us/azure/postgresql/concepts-pricing-tiers#storage-auto-grow